src/components/NewCar/NewCar.js

```diff
@@ -0,0 +1,176 @@
+import { useState } from 'react';
+import { Link } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+import { getDownloadURL, ref, uploadBytes } from 'firebase/storage';
+import { v4 } from 'uuid';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faXmark } from '@fortawesome/free-solid-svg-icons';
+import storage from './firebase';
+import { addCar } from '../../redux/reducers/cars';
+
+const logo = './logo.svg';
+
+function NewCar() {
```

Comment:
- [ ]  Please, kindly refactor this function in your project to ES6 arrow function, so your code complies with the latest standards.

Refactoring is easy:

```javascript
const NewCar= () => {
  // ...
}
```

## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions. This means, almost nobody uses it anymore, so much that it's considered non-best practice.